Misc/NEWS.d/next/Library/2023-11-24-19-08-50.gh-issue-112343.RarGFC.rst

```diff
@@ -0,0 +1 @@
+Switch to :mod:`tokenize` for convenience variables to avoid replacing strings.
```

Comment:
```suggestion
Switch to :mod:`tokenize` for pdb convenience variables to avoid replacing strings.
```

Reply:
I'd actually probably avoid mentioning tokenize at all in this news entry — news entries should generally discuss user-visible behaviour changes rather than the implementation details of how those changes have been achieved

Reply:
```suggestion
Improve handling of pdb convenience variables to avoid replacing string contents.
```